spec/support/database_cleaner.rb

```diff
@@ -2,6 +2,11 @@
 
   config.before(:suite) do
     DatabaseCleaner.clean_with(:truncation)
+    begin
+      DatabaseCleaner.start
+    ensure
+      DatabaseCleaner.clean
+    end
```

Comment:
You may want to have a more robust database cleaner setup if you start to use JS or JQuery. Check out this blog post: http://www.virtuouscode.com/2012/08/31/configuring-database_cleaner-with-rails-rspec-capybara-and-selenium/